python/scripts/report_on_repository_standards.py

```diff
@@ -57,19 +58,17 @@ def __parse_args(args):
 
 
 def main():
-    # Add arguments from the command line
     args = __parse_args(__add_arguments())
-
-    # Fetch all repositories in the org
     repos = GithubService(
         args.oauth_token, args.org).fetch_all_repositories_in_org()
-
-    # Generate GitHub standards report for each repository in the org
     repo_reports = [RepositoryReport(repo).output for repo in repos]
 
-    # Send the reports to the operations-engineering-reports API
-    reports_service(args.url, args.endpoint, args.api_key)\
-        .override_repository_standards_reports(repo_reports)
+    #  The database can't handle more than 100 at a time
+    # so we need to chunk the list into 100s.
+    api = reports_service(args.url, args.endpoint, args.api_key)
+    for i in range(0, len(repo_reports), 100):
```

Comment:
Could we push this logic into `reports_service.override_repository_standards_reports()`?

Reply:
Good idea. I've now moved the mode to the reports service.